FWCore/Framework/interface/maker/Worker.h

```diff
@@ -241,6 +241,9 @@ namespace edm {
 
     virtual bool hasAccumulator() const = 0;
 
+    // Used in PuttableProductResolver
+    edm::WaitingTaskList* waitingTaskList() { return &waitingTasks_; }
```

Comment:
Given this can never return a null, maybe it would be better to return `edm::WaitingTaskList&`

Reply:
Thanks, done